nixos/modules/profiles/all-hardware.nix

```diff
@@ -58,15 +58,7 @@ in
       # Hyper-V support.
       "hv_storvsc"
     ] ++ lib.optionals pkgs.stdenv.hostPlatform.isAarch [
-      # Most of the following falls into two categories:
-      #  - early KMS / early display
-      #  - early storage (e.g. USB) support
-
```

Comment:
I know I'm a bit late, but this is still true and needed.

Reply:
That comment got copy/pasted into every branch for some reason, I just removed the extra copy.